lib/airbrake/logger/logger_ext.rb

```diff
@@ -75,4 +75,20 @@ def airbrake_severity(severity)
        'critical'
      end).freeze
   end
+
+  # @!macro helper_method
+  #   @note We define this helper instead of using instance variable to avoid
+  #     Ruby's warnings about uninitialized ivars.
+
+  ##
+  # @macro helper_method
+  def current_airbrake_notifier
+    airbrake_notifier || Airbrake[:default]
+  end
+
+  ##
+  # @marcro helper_method
+  def current_airbrake_severity
```

Comment:
I could not come up with a better name. These shenanigans are needed to eliminate ruby warnings about unassigned instance variables that used to be in the initialise method.